gulpfile.js

```diff
@@ -0,0 +1,39 @@
+var gulp = require('gulp'),
```

Comment:
Good question. We can't use it fully because it requires ES6 and I don't think it's worth to implement Babel just for the gulpfile (wow, that would be beautifully over-engineered). And I think we shouldn't depend on node.js version being > 4 before we have it updated on all devboxes.